### PR TITLE
Add scopes to run tc-admin apply for proj-fuzzing

### PR DESCRIPTION
**Pull request by jschwartzentruber**
Also move some secret scopes to master branch only, where they are only needed for CD not CI.

### CODE REVIEW

**Comment by La0** [comment]
LGTM :+1:

File: config/projects/fuzzing.yml [hunk 1]
```diff
@@ -46,14 +46,20 @@ fuzzing:
...
+        - assume:hook-id:project-fuzzing/*
+        - auth:create-role:hook-id:project-fuzzing/*
+        - auth:update-role:hook-id:project-fuzzing/*
+        - hooks:modify-hook:project-fuzzing/*
+        - hooks:trigger-hook:project-fuzzing/*
+        - queue:scheduler-id:-
+        - secrets:get:project/fuzzing/credstash-aws-auth
+        - secrets:get:project/fuzzing/deploy-*
+        - worker-manager:manage-worker-pool:proj-fuzzing/*
+        - worker-manager:provider:community-tc-workers-*
```
Comment (petemoore) [inline config/projects/fuzzing.yml]: @imbstack is line 63 ok? I noticed the scope isn't specific to the fuzzing project, so I wanted to check with someone that knows more about worker manager. Thanks!

Comment (imbstack) [reply config/projects/fuzzing.yml]: That scope is required to create/update/delete worker pools that use those providers so if that's the intention to allow that here then it makes sense to me!

**Comment by djmitche** [comment]
Applicated!

**Pull request merged**